app/views/layouts/application.html.slim

```diff
@@ -48,6 +48,36 @@ html dir="ltr" lang="en-GB" class="govuk-template"
         div role="navigation"
           = yield :back
 
+      - if beta_pilot_url? && FeatureFlags.enabled?(:public_beta, without_session_data: true)
+       - if controller_name == "estimates"
```

Comment:
This is another not very ideal check. It just happens to be the case that all `estimates` pages want full width banners and all other pages want 2-thirds width, but checking the controller name isn't very easy to read, there are lots of structural refactors we could make where this will break, and when a different page wants a different width we won't thank ourselves for not having made it easy for ourselves. Also, this is hugely repetitive - we repeat 12 lines of code verbatim thanks to this `if` statement.

Here's what I suggest. As per the guidance, this component should always appear directly above the h1, right? So let's insert the banner into `shared/_heading.html.slim`. On 2-thirds-width pages that will already be rendered inside `govuk-grid-column-two-thirds` divs, so it will always be the width we want.

The only gotcha is that we don't always use the `shared/heading` component yet - most notably on single-question pages like `provider_users/show`, where we build the h1 element from the legend of the field component. In those cases, I think the solution is now to render that component in all cases (it's just that on screens like that we _wouldn't_ pass in `header_text` as a param). That would also tidy up the way that in a lot of cases we're manually rendering `shared/household_tag` and `shared/partner_tag`, while in some cases we're relying on `shared/heading` to do it for us. If we call `shared/heading` on every screen, we can _stop_ manually rendering those tags. So then `heading` becomes more useful and our screens all become more consistent.